becquerel/core/plotting.py

```diff
@@ -532,3 +466,78 @@ def linthreshy(self):
         min_ind = np.argmin(np.abs(self._ydata[self._ydata != 0]))
         delta_y = np.abs(self._ydata - self._ydata[min_ind])
         return np.min(delta_y[delta_y > 0])
+
+
+def get_channel_edges(channels):
```

Comment:
@cosama why did you remove the `staticmethods` in favor of funcs? Is there a style guideline regarding this modification?

Reply:
Yeah it is in the google style guide to rather use module methods. Not sure why though. Do you have any ideas?

Reply:
Nope. Let’s follow the google style guide on this. Thanks!